staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -207,11 +207,11 @@ func (c completedConfig) newFromExistingServer(genericServer *genericapiserver.G
 
 // AddAPIService adds an API service.  It is not thread-safe, so only call it on one thread at a time please.
 // It's a slow moving API, so its ok to run the controller on a single thread
-func (s *APIAggregator) AddAPIService(apiService *apiregistration.APIService) {
+func (s *APIAggregator) AddAPIService(apiService *apiregistration.APIService, destinationHost string) {
 	// if the proxyHandler already exists, it needs to be updated. The aggregation bits do not
 	// since they are wired against listers because they require multiple resources to respond
 	if proxyHandler, exists := s.proxyHandlers[apiService.Name]; exists {
-		proxyHandler.updateAPIService(apiService)
+		proxyHandler.updateAPIService(apiService, destinationHost)
 		return
 	}
 
```

Comment:
if we allow `apiService.Spec.Group == ""` (which it appears we do), we should make sure we don't ever have a case where `apiService.Spec.Group == ""` and `apiService.Spec.Version != "v1"`

as it is now, that could give us a proxyPath of `/apis//vX`